screenpipe-vision/Cargo.toml

```diff
@@ -82,15 +83,15 @@ path = "examples/websocket.rs"
 
 [target.'cfg(target_os = "windows")'.dependencies]
 windows = { version = "0.58", features = ["Graphics_Imaging", "Media_Ocr", "Storage", "Storage_Streams"] }
-xcap = "0.0.12"
+#xcap = "0.0.12"
 
 [target.'cfg(target_os = "macos")'.dependencies]
 libc = "0.2"
-xcap = "0.0.13" # memory leak fix, also for some reason 0.0.13 doesn't work on windows
+#xcap = "0.0.13" # memory leak fix, also for some reason 0.0.13 doesn't work on windows
 
 [target.'cfg(target_os = "linux")'.dependencies]
 libc = "0.2"
-xcap = "0.0.12"
```

Comment:
https://crates.io/crates/xcap

seems he released 0.0.14

@FractalFir 

are you in hurry for merge? i'd like to merge but still trying to find anyone else to test it out, maybe this weekend

Reply:
> seems he released 0.0.14

That's great, since this means you can just update xcap version for Linux, and keep everything else unchanged. I will try and see if that works, and update the PR. 

> are you in hurry for merge? i'd like to merge but still trying to find anyone else to test it out, maybe this weekend

No. While I would like to know if the fixes fully work, I can easily wait for a couple more days. I would prefer if the changes got fully tested before a merge, since I don't want to have to reverse changes. So, waiting a bit for a proper test is  fine by me.